contrib/lorri.nix

```diff
@@ -0,0 +1,38 @@
+{ config, systemd, ... }:
+
+let
+  lorri = import
+    (fetchTarball "https://github.com/target/lorri/tarball/rolling-release")
+    { };
+  socketUnit = "lorri";
+  socketPath = "lorri/daemon.socket";
+in {
+  config = {
+    systemd.user.sockets.${socketUnit} = {
+      description = "Socket for Lorri Daemon";
+      enable = true;
+      wantedBy = [ "sockets.target" ];
+      socketConfig = {
+        ListenStream = "%t/${socketPath}";
+        RuntimeDirectory = "lorri";
+      };
+    };
+
+    systemd.user.services.lorri = {
```

Comment:
The `PATH` that I end up with for this doesn't have access to `nix-instantiate`, so `lorri` doesn't actually work.